Makefile

```diff
@@ -0,0 +1,135 @@
+# ----------------------------------------------------------------------
+#  Development Noetic
+# ----------------------------------------------------------------------
+
+#: Builds a Docker image from the current Dockerfile file
+noetic.build:
+	@docker build -t ros:noetic -f Dockerfile.noetic .
+
+
+#: Create Docker container
+noetic.create: 
+	@docker run \
+		-it -d \
+		-v ${shell pwd}/catkin_home:/catkin_home \
+		--network host \
+		--name ros-noetic \
+		ros:noetic
+
+#: Start the container in background
+noetic.up:
+	@docker start ros-noetic
+
+#: Stop the container
+noetic.down:
+	@docker stop ros-noetic
+
+#: Restarts the container
+noetic.restart:
+	@docker restart ros-noetic
+
+#: Shows the logs of the ros-noetic service container
+noetic.logs:
+	@docker logs --tail 50 ros-noetic
+
+#: Fires up a bash session inside the ros-noetic service container
+noetic.shell:
+	@docker exec -it ros-noetic bash
+
+#: Remove ros-noetic container. 
+noetic.remove: noetic.down
+	@docker container rm ros-noetic
+
+# ----------------------------------------------------------------------
+#  Development Melodic
+# ----------------------------------------------------------------------
+
+#: Builds a Docker image from the current Dockerfile file
+melodic.build:
+	@docker build -t ros:melodic -f docker/melodic/Dockerfile .
+melodic.speech.build:
+	@docker build -t ros:melodic-speech -f docker/melodic/Dockerfile.speech .
```

Comment:
Note: I would really like for the documentation to mention that we have to define the context with -f

Reply:
I will consider this once i start working in the wiki entry.